<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SimmonGuiWindow</class>
 <widget class="QMainWindow" name="SimmonGuiWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>942</width>
    <height>564</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="windowIcon">
   <iconset resource="res.qrc">
    <normaloff>:/main/sm_icon.png</normaloff>:/main/sm_icon.png</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QPushButton" name="watcherExpandBtn">
      <property name="styleSheet">
       <string notr="true">background: rgba(100, 100, 100,0); border: 1px solid; border-radius: 3px; border-color: rgb(146, 146, 146); margin: 1px;</string>
      </property>
      <property name="text">
       <string>Background Watchman</string>
      </property>
      <property name="icon">
       <iconset resource="res.qrc">
        <normaloff>:/main/keyboard_arrow_down-24px.svg</normaloff>
        <normalon>:/main/keyboard_arrow_up-24px.svg</normalon>:/main/keyboard_arrow_down-24px.svg</iconset>
      </property>
      <property name="checkable">
       <bool>true</bool>
      </property>
      <property name="checked">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QWidget" name="watcherWidget" native="true">
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <property name="leftMargin">
        <number>6</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>6</number>
       </property>
       <property name="bottomMargin">
        <number>6</number>
       </property>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item alignment="Qt::AlignLeft">
          <widget class="QLabel" name="watcherStatusLabel">
           <property name="text">
            <string>Status:</string>
           </property>
          </widget>
         </item>
         <item alignment="Qt::AlignLeft">
          <widget class="QLabel" name="watcherStatusText">
           <property name="text">
            <string>not detected</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QPushButton" name="watcherUpdateBtn">
           <property name="text">
            <string>Update</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="watcherStartBtn">
           <property name="text">
            <string>Start</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="watcherStopBtn">
           <property name="text">
            <string>Stop</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QTabWidget" name="profileTabWidget">
      <property name="currentIndex">
       <number>-1</number>
      </property>
      <property name="tabsClosable">
       <bool>true</bool>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>942</width>
     <height>18</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionImport"/>
    <addaction name="actionLog"/>
    <addaction name="actionAppLog"/>
    <addaction name="actionExit"/>
   </widget>
   <addaction name="menuFile"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionExit">
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
  <action name="actionLog">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/main/filter_none-24px.svg</normaloff>:/main/filter_none-24px.svg</iconset>
   </property>
   <property name="text">
    <string>View Log</string>
   </property>
  </action>
  <action name="actionImport">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/main/description-24px.svg</normaloff>:/main/description-24px.svg</iconset>
   </property>
   <property name="text">
    <string>Import Profile</string>
   </property>
  </action>
  <action name="actionAppLog">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/main/filter_none-24px.svg</normaloff>:/main/filter_none-24px.svg</iconset>
   </property>
   <property name="text">
    <string>View App Log</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="res.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>actionExit</sender>
   <signal>triggered()</signal>
   <receiver>SimmonGuiWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>426</x>
     <y>272</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
